Makefile

```diff
@@ -72,6 +72,9 @@ image: imagebuilder
 test-e2e:
 	hack/test-e2e.sh
 
+test-unit:
+	@go test -v $(PKGS)
```

Comment:
I think it would be better to test only the directories `cmd` and `pkg`, rather than trying to figure out where the source code might be.  It should be fairly rare to add new source code directories

Reply:
updated to be explicitly `pkg` and `cmd`